src/app/constructor/dashboard/dashboard-constructor.component.ts

```diff
@@ -10,38 +10,46 @@ import { Device } from '../../shared/devices/device.model';
 export class DashboardConstructorComponent {
     @Input() canBeActive: boolean;
     @Input() default: string;
+    @Input() dashboardSubView: DashboardViewInfoDto;
```

Comment:
I can't agree with you because in parent constructor.component we should pass mapView and dashboardView into its children component and we should differ them to not be confused. It seems that in future we have to  combine map-constructor and dashboard into one because it's one essence.